### PR TITLE
new gateway has been added

### PR DESCRIPTION
**Pull request by hsyntnc**

### CODE REVIEW

**Comment by onurozgurozkan** [comment]
👍

File: readme.txt [hunk 1]
```diff
@@ -90,6 +90,7 @@ Gateways:
 * cpsms.dk
 * bulksmshyderabad.co.in
 * ozioma.net
+* Bulutfon.com
```
Comment (mostafasoufi) [inline readme.txt:93]: This is premium gateway list.

Please remove it.

Comment (onurozgurozkan) [inline readme.txt:93]: Dear @mostafasoufi,

First off all thank you very much for this great plugin.

I'm the co-founder of Bulutfon. If it isn't problem for you, you can also add Bulutfon to premium list. We'll send email to our clients that `wp-sms` plugin and `wp sms professional package` are both integreted with Bulutfon. If they need some more feature like Buddy press integration, they may choice `wp sms professional package` . 

Regards.

Comment (mostafasoufi) [inline readme.txt:93]: Hello,

So, give me an access to your service provider to check it.

Regards

Comment (onurozgurozkan) [inline readme.txt:93]: Hi,

Can we send the information to hi at veronalabs dot com email? I don't want to share token information publicly. 

Regards.

Comment (mostafasoufi) [inline readme.txt:93]: Yes.you can.

**Pull request merged**